daemon/graphdriver/aufs/aufs.go

```diff
@@ -360,10 +364,20 @@ func (a *Driver) Diff(id, parent string) (archive.Archive, error) {
 }
 
 func (a *Driver) applyDiff(id string, diff archive.Reader) error {
-	return chrootarchive.UntarUncompressed(diff, path.Join(a.rootPath(), "diff", id), &archive.TarOptions{
+	dir := path.Join(a.rootPath(), "diff", id)
```

Comment:
There are 18 mentions of `path.*` in this file. That change needs to be in another PR.